lighthouse-core/config/budget.js

```diff
@@ -5,10 +5,36 @@
  */
 'use strict';
 
+/**
+ * @param {unknown} arr
+ * @return {arr is Array<Record<string, unknown>>}
+ */
+function isArrayOfUnknownObjects(arr) {
```

Comment:
i'd love to share more of this logic across the config files

Reply:
> i'd love to share more of this logic across the config files

agreed!